packages/xo-server/src/api/host.js

```diff
@@ -1,17 +1,22 @@
 import { format, JsonRpcError } from 'json-rpc-peer'
+import { satisfies as versionSatisfies } from 'semver'
 
 // ===================================================================
 
 export async function set({
   host,
-
+  iscsiIqn,
   multipathing,
   name_label: nameLabel,
   name_description: nameDescription,
 }) {
   host = this.getXapiObject(host)
 
   await Promise.all([
+    versionSatisfies(host.version, '>=7.5.0')
+      ? iscsiIqn !== undefined && host.set_iscsi_iqn(iscsiIqn)
+      : iscsiIqn !== undefined &&
+        host.update_other_config('iscsi_iqn', iscsiIqn),
```

Comment:
For the `other_config` field, the default value may simply be to remove the `iscsi_iqn` entry.